server/routes/managedPageRouter.ts

```diff
@@ -36,6 +36,11 @@ export default function whatsNewRouter(services: Services): Router {
     getFrontendComponents(services, config.apis.frontendComponents.latest),
     managedPageController.displayManagedPage('cookies-policy'),
   )
+  get(
```

Comment:
I've included a test url temporarily so Dom can test without affecting the real Contentful pages.
Will remove in another PR before pushing to prod